src/modules/mc_rate_control/MulticopterRateControl.cpp

```diff
@@ -147,12 +145,18 @@ MulticopterRateControl::Run()
 	vehicle_angular_velocity_s angular_velocity;
 
 	if (_vehicle_angular_velocity_sub.update(&angular_velocity)) {
+
+		// grab corresponding vehicle_angular_acceleration immediately after vehicle_angular_velocity copy
+		vehicle_angular_acceleration_s v_angular_acceleration{};
+		_vehicle_angular_acceleration_sub.copy(&v_angular_acceleration);
```

Comment:
Minor implementation detail but what about using a `SubscriptionData` and directly fetching the vector using:
`const Vector3f angular_accl{_vhicle_angular_acceleration_sub.get().xyz};` ?

Reply:
It's pretty minor, but there's no need to store the data because every iteration grabs a new one.